cmd/lakectl/cmd/fs_download.go

```diff
@@ -79,6 +84,7 @@ var fsDownloadCmd = &cobra.Command{
 					After:        (*apigen.PaginationAfter)(swag.String(after)),
 					Prefix:       (*apigen.PaginationPrefix)(remote.Path),
 					UserMetadata: swag.Bool(true),
+					Presign:      swag.Bool(false), // faster listing without, we use stat object later if we need it
```

Comment:
What's the default? I'm wondering if this is for being more explicit with the default, or for changing the value that passed. If it's the latter, don't you worry from degraded performance in some other flow?